frontend/components/utils/BidInput.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+    <div class="w-full inline-block relative flex-shrink-0">
+        <Tooltip :visible="!!error" placement="topLeft" :title="error" class="items-center">
+            <div v-if="!amountToBid" class="Overlay TotalPrice">
+                <format-currency v-if="totalPrice && !disabled" :value="totalPrice" />
+                <span v-else class="opacity-50">UNKNOWN</span>
```

Comment:
Let's make it the same as in other table fields:
```suggestion
                <span v-else class="opacity-50">Unknown</span>
```